scripts/ayp.coffee

```diff
@@ -335,8 +339,11 @@ class AYPStrip
 
           @compositeImage frame, char, left, top
 
+          isKenny = char.matchchar.match('default') == null
+          scrambleLines = scrambleLines.concat(line.concat(isKenny))
```

Comment:
Same feedback as above, but also please drop a comment explaining the transform in either place.